backend/api/Models/Concepts/ResearchFile/ResearchFileModel.cs

```diff
@@ -14,7 +14,7 @@ public class ResearchFileModel : FileModel
         /// <summary>
         /// get/set - A list of research property relationships.
         /// </summary>
-        public IList<ResearchFilePropertyModel> ResearchProperties { get; set; }
+        public override IList<ResearchFilePropertyModel> FileProperties { get; set; }
```

Comment:
research file properties have some additional fields not part of the base FileProperties entity so overriding here.